src/components/QuestDetails/components/Reward/index.tsx

```diff
@@ -40,8 +44,24 @@ export default function Reward({ reward, i18n }: RewardProps) {
       <div className={styles.numToClaim}>{`+${formattedNumToClaim}`}</div>
     )
   }
+
+  const onViewRewardClick = () => {
+    if (!reward.marketplaceUrl) {
+      return
+    }
+
+    window.open(reward.marketplaceUrl, '_blank')
+  }
+
   return (
     <div key={reward.title} className={styles.rewardContainer}>
+      {reward.isClaimed ? (
+        <div className={styles.isClaimed}>{i18n.claimed}</div>
+      ) : null}
+      <div className={styles.viewRewardContainer} onClick={onViewRewardClick}>
```

Comment:
can you make this an a tag and set the src to `reward.marketplaceUrl`?

Reply:
Yup can do 🙌

Reply:
Sorted 🙌